source/_docs/z-wave/adding.markdown

```diff
@@ -77,11 +82,12 @@ Secure devices require additional bandwidth, and too many secure devices can slo
 
 To remove (exclude) a Z-Wave device from your system:
 
-1. Go to the Z-Wave control panel in the Home Assistant frontend
+1. Go to the Z-Wave control panel in the Home Assistant frontend. (Home Assistant > Configuration > Integrations > Configure)
 2. Click the **Remove Node** button in the *Z-Wave Network Management* card - this will place the controller in exclusion mode
 3. Activate your device to be excluded by following the instructions provided with the device
-4. The device will now be removed, but that won't show until you restart Home Assistant
+4. The device should now be removed, but that won't show until you restart Home Assistant. Look for confirmation signal on the device if available, or confirm on the Home Assistant logs.
 5. Run a *Heal Network* so all the other nodes learn about its removal
+6. Make sure to Restart Home Assistant. Done through **Home Assistant > Configuration > Server Controls**
```

Comment:
Why would we need to restart?

Reply:
Removing this line then if you believe it won't be required. I had just seen it referred as a common step for most configuration updates.

Reply:
Configuration via the UI doesn't require restart.